shared/src/common/enums/api/worker-api-path.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum WorkerApiPath {
```

Comment:
```suggestion
enum EAMApiPath {
```
don't forget about file name